src/plugins/auto_batch/tests/unit/plugin_get_property_test.cpp

```diff
@@ -63,6 +63,9 @@ const std::vector<get_property_params> get_property_params_test = {
     get_property_params{ov::device::priorities.name(), true},
     get_property_params{ov::cache_dir.name(), true},
     get_property_params{ov::hint::performance_mode.name(), true},
+    get_property_params{ov::enable_profiling.name(), false},
+    get_property_params{ov::num_streams.name(),true}
```

Comment:
```suggestion
    get_property_params{ov::num_streams.name(),false}
```

Reply:
You need mock the function "get_property" in setup, "get_property_params{ov::num_streams.name(),false}" no throw exception